GenerationExpansionPlanning/src/relaxation.jl

```diff
@@ -1,16 +1,43 @@
 export relaxation_iteration
 """
-    relaxation_iteration(data::ExperimentData, k=2)::(ExperimentData, Vector{Set{Symbol}})
-
     Create clusters and returns merged Dataframes
 """
-function relaxation_iteration(data::ExperimentData, dendogram::Hclust, k=2)::Tuple{ExperimentData, Vector{Set{Symbol}}}
+function relaxation_iteration(data::ExperimentData, k::Integer=2)::Tuple{ExperimentData, Vector{Set{Symbol}}}
```

Comment:
This should keep dendogram, because experiment calculates the dendogram before doing any relaxation

Reply:
btw it is dendRogram, not demdogram